doc/faq.md

```diff
@@ -27,6 +27,8 @@ Yes, so long as Neos's [Mod & Plugin Policy] is followed.
 ## Will people know I'm using mods?
 
 - By default, NeosModLoader does not do anything identifiable over the network. You will appear to be running the vanilla Neos version to any component that shows your version strings or compatibility hash.
+- It is still technically possible for a user to determine if you are running NeosModLoader or a specific mod, although this requires a specific LogiX setup.
```

Comment:
Add a link to #64 so that interested users can track the issue progress. Also, consider removing much (or all) of the explanation as interested users can read that in #64.